### PR TITLE
Changes in the system testing container

### PR DESCRIPTION
**Pull request by pro-akim**
|Related issue|
|-------------|
|    https://github.com/wazuh/wazuh-qa/issues/4682         |

## Description

Previous containers had performance problems. It could be related to its size (around 9gbs). 
It will revert changes and It will be important to run the tests using package method:

```
ansible-playbook -i inventory.yml playbook.yml --extra-vars='{"package_repository":"packages", "repository": "4.x", "package_version": "4.4.0", "package_revision": "1"}'
```

### Updated

- Reverse to previous containers
- Provisioning by repository is deprecated

## Testing performed

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @pro-akim  (Developer)  |           | ⚫⚫⚫ |[:green_circle:](https://github.com/wazuh/wazuh-qa/files/13334158/1.zip)[:green_circle:](https://github.com/wazuh/wazuh-qa/files/13334159/2.zip)[:green_circle:](https://github.com/wazuh/wazuh-qa/files/13334160/3.zip)|         |         | Nothing to highlight |
| @user (Reviewer)   |           | ⚫⚫⚫ | :no_entry_sign: :no_entry_sign: :no_entry_sign:  |        |         | Nothing to highlight |

### CODE REVIEW

**Comment by pro-akim** [comment]
## Update:

Linting errors are not related with recent changes.

**Review by Rebits — CHANGES_REQUESTED** [review]
Some changes are required:
- [x] Deprecate temporally the source installation
- [x] Address the instability in certain Docker environments in an issue

#### Source Installation Deprecation:

If the source installation always fail from 4.7.0 due to the gcc image, let's consider deprecating it from the playbook temporary. This would streamline the deployment process, allowing only the use of the packages option.

#### Container Sensitivity:

Regarding [this comment](https://github.com/wazuh/wazuh-qa/issues/4682#issuecomment-1804207550), it's clear that Wazuh, not the tests, exhibits sensitivity to container environments. Notably, Syscheck alerts are selectively generated based on the environment. Proposing to open a new issue to thoroughly investigate this behavior and ensure Wazuh's seamless operation across various environments. The goal is to make Wazuh adaptable, regardless of image size.

**Comment by pro-akim** [comment]
## Update

A new issue has been opened: https://github.com/wazuh/wazuh/issues/20191 regarding the sensitivity of Wazuh in certain containers.

**Comment by pro-akim** [comment]
## Update
Install by repository has been deprecated.
3 different environments were tested and reports are attached in the [general report](https://github.com/wazuh/wazuh-qa/pull/4686#issue-1986043340)


Linting errors are not related to changed code.

**Review by Rebits — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fix of system test instability ([#4686](https://github.com/wazuh/wazuh-qa/pull/4686)) \- (Tests)
```
[the remark this answers is not in the document]
Comment (pro-akim) [reply CHANGELOG.md]: Done in https://github.com/wazuh/wazuh-qa/pull/4686/commits/d98e795ddff87edce5f4bec2140d393a501dfb9b

**Review by Rebits — APPROVED** [review]
LGTM

**Pull request merged**